lib/repo/git_repository.rb

```diff
@@ -2,6 +2,7 @@
 require 'gitolite'
 require 'digest/md5'
 require 'rubygems'
+require "git"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.